extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1060-view-exceptions_LMSSection.sql

```diff
@@ -3,7 +3,7 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
```

Comment:
These fixes mirror those in Luis' branch

Reply:
Actually Luis's branch has `lmsx` for the schema on the views and stored procedures - matching the extension instead of the LMS Tooolkit.